### PR TITLE
[stable/jenkins] deprecate chart

### PR DESCRIPTION
**Pull request by torstenwalter**
This chart was moved to a new location: https://github.com/jenkinsci/helm-charts

Part-of: #21103
Part-of: #23562
Part-of: jenkinsci/helm-charts#4

Should be merged after https://github.com/jenkinsci/helm-charts/pull/1

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/jenkins/README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# ⚠️ DEPRECATED
+
+This chart was moced to https://github.com/jenkinsci/helm-charts
```
Comment (wmcdona89) [inline stable/jenkins/README.md]: typo "moved"

**Comment by torstenwalter** [comment]
This is good to go:

```console
$ helm repo add jenkinsci https://charts.jenkins.io/
$ helm search repo jenkinsci
"jenkinsci" has been added to your repositories
NAME             	CHART VERSION	APP VERSION	DESCRIPTION                                       
jenkinsci/jenkins	2.6.0        	lts        	Open source continuous integration server. It s...
```

**Review by scottrigby — APPROVED** [review]
🏆

**Pull request merged**